src/components/SignUp.js

```diff
@@ -1,74 +1,164 @@
+                            <option key='blankChoice' hidden value/>
+                            <option value="Male" onChange={(e) => {setGender(e.target.value)}}>Male</option>
+                            <option value="Female" onChange={(e) => {setGender(e.target.value)}}>Female</option>
+                        </Form.Select>
+                    </Form.Group>
+
+                    <Form.Group className="mb-3" controlId="formBasicUsername">
+                        <Form.Label>Username</Form.Label>
+                        <Form.Control type="email" placeholder="Create Username" onChange={(e) => {setUsername(e.target.value)}}/>
+                    </Form.Group>
+
+                    <Form.Group className="mb-3" controlId="formBasicEmail">
                         <Form.Label>Email address</Form.Label>
-                        <Form.Control type="email" placeholder="Enter email" />
+                        <Form.Control type="email" placeholder="Enter email" onChange={(e) => {setEmail(e.target.value)}}/>
                         <Form.Text className="text-muted">
                             We'll never share your email with anyone else.
                         </Form.Text>
                     </Form.Group>
-  
-                    <Form.Group className="mb-3" controlId="form-Basic-Password">
+
+                    <Form.Group className="mb-3" controlId="formBasicPassword">
                         <Form.Label>Create Password</Form.Label>
                         <Form.Control
                             type="password"
-                            placeholder="Password"/>
-                        <Form.Text id="password-Help-Block" muted>
+                            placeholder="Password" onChange={(e) => {setPassword(e.target.value)}}/>
+                        <Form.Text id="passwordHelpBlock" muted>
                             Your password must be 8-20 characters long, contain letters and numbers, and
                             must not contain spaces, special characters, or emoji.
                         </Form.Text>
                     </Form.Group>
-  
-                    <Form.Group className="mb-3" controlId="form-Basic-Password">
+
+                    <Form.Group className="mb-3" controlId="formBasicRepeatPassword">
                         <Form.Label>Submit Password Again</Form.Label>
                         <Form.Control
                             type="password"
-                            placeholder="Password"/>
+                            placeholder="Password" onChange={(e) => {setRepeatPassword(e.target.value)}}/>
                     </Form.Group>
-  
 
-                    <Form.Group className="mb-3" controlId="submit-form">
-                        <Button variant="outline-info" type="submit">
+                    <Form.Group className="mb-3" controlId="submitform">
+                        <Button variant="outline-info" to='/auth/signin' type="submit" onClick={handleClick}>
```

Comment:
Same thing here, does to `to` work here?

Reply:
I forgot to delete it before.